### PR TITLE
applying proofreading changes to c12

### PR DESCRIPTION
**Pull request by jannes-m**

### CODE REVIEW

**Review by Nowosad — APPROVED** [review]

File: 12-spatial-cv.Rmd [hunk 1]
```diff
@@ -274,7 +274,7 @@ It acts as a 'meta-package', providing a unified interface to popular supervised
 The standardized **mlr3** interface is based on eight 'building blocks'.
 As illustrated in Figure \@ref(fig:building-blocks), these have a clear order.
 
-(ref:building-blocks) Basic building blocks of the mlr3 package. Source: @bischl_applied_2024. (Permission to reuse this figure was kindly granted.)
+(ref:building-blocks) Basic building blocks of the mlr3 package. @bischl_applied_2024. Permission to reuse this figure was kindly granted.
```
Comment (Robinlovelace) [inline 12-spatial-cv.Rmd:277]:
```suggestion
(ref:building-blocks) Basic building blocks of the mlr3 package @bischl_applied_2024. Permission to reuse this figure was kindly granted.
```

File: 12-spatial-cv.Rmd [hunk 2]
```diff
@@ -568,7 +568,7 @@ To make the performance estimation processing chain even clearer, let us write d
 1. Performance level (upper left part of Figure \@ref(fig:inner-outer)) - split the dataset into five spatially disjoint (outer) subfolds
 1. Tuning level (lower left part of Figure \@ref(fig:inner-outer)) - use the first fold of the performance level and split it again spatially into five (inner) subfolds for the hyperparameter tuning. 
 Use the 50 randomly selected hyperparameters\index{hyperparameter} in each of these inner subfolds, i.e., fit 250 models
-1. Performance estimation - Use the best hyperparameter combination from the previous step (tuning level) and apply it to the first outer fold in the performance level to estimate the performance (AUROC\index{AUROC})
+1. Performance estimation - use the best hyperparameter combination from the previous step (tuning level) and apply it to the first outer fold in the performance level to estimate the performance (AUROC\index{AUROC})
```
Comment (Robinlovelace) [inline 12-spatial-cv.Rmd:571]:
```suggestion
1. Performance estimation: use the best hyperparameter combination from the previous step (tuning level) and apply it to the first outer fold in the performance level to estimate the performance (AUROC\index{AUROC})
```

Or --- m-dash, not the "-" dash.

**Review by Robinlovelace — APPROVED** [review]
Definite improvements in there. I've made a couple of other suggestions.

File: 12-spatial-cv.Rmd (hunk 1 above)
Comment (jannes-m) [reply 12-spatial-cv.Rmd:277]: yeah, I was wondering about this as well, thanks!

File: 12-spatial-cv.Rmd (hunk 2 above)
Comment (jannes-m) [reply 12-spatial-cv.Rmd:571]: looks good to me, thanks

**Pull request merged**